src/group_theory/specific_groups/alternating.lean

```diff
@@ -100,4 +146,40 @@ closure_eq_of_le _ (λ σ hσ, mem_alternating_group.2 hσ.sign) $ λ σ hσ, be
     (ih _ (λ g hg, hl g (list.mem_cons_of_mem _ (list.mem_cons_of_mem _ hg))) hn),
 end
 
+lemma is_three_cycle.top_le_alternating_normal_closure (h5 : 5 ≤ fintype.card α)
+  {f : perm α} (hf : is_three_cycle f) :
+  ⊤ ≤ normal_closure ({⟨f, hf.mem_alternating_group⟩} : set (alternating_group α)) :=
```

Comment:
Would it be better to phrase this as `... = ⊤`, and then start the proof with `rw eq_top_iff`?

Reply:
The applications of all these lemmas actually are simpler using `le_trans` than with rewriting.

Reply:
I've tried both ways, and doing everything with equality takes more steps, although the statements look nicer by themselves. I'm inclined to go with the way that serves the overall proof of simplicity better, because I doubt anyone's going to use these lemmas as such for anything else (or if they do, they'll need to change things anyway).

Reply:
@awainverse I think @tb65536 is suggesting that you only change the statement into `normal_closure ... = \top` and then in the first step of the proof, use `rw eq_top_iff` to reduce it to your original statement. So the rest of the proof is just your old proof.
In principal, it should make the lemma easier to apply.